cli/bench_command.go

```diff
@@ -66,305 +62,497 @@ type benchCmd struct {
 	deDuplicationWindow  time.Duration
 	retries              int
 	retriesUsed          bool
+	ack                  bool
 }
 
 const (
-	DefaultDurableConsumerName string = "natscli-bench"
-	DefaultStreamName          string = "benchstream"
-	DefaultBucketName          string = "benchbucket"
+	BenchDefaultDurableConsumerName string = "nats-bench"
+	BenchDefaultStreamName          string = "benchstream"
+	BenchDefaultBucketName          string = "benchbucket"
+	BenchDefaultServiceName         string = "nats-bench-service"
+	BenchDefaultServiceVersion      string = "1.0.0"
+	BenchTypeCorePub                string = "pub"
+	BenchTypeCoreSub                string = "sub"
+	BenchTypeServiceRequest         string = "request"
+	BenchTypeServiceServe           string = "reply"
+	BenchTypeJSPub                  string = "jspub"
+	BenchTypeJSOrdered              string = "jsordered"
+	BenchTypeJSConsume              string = "jsconsume"
+	BenchTypeJSFetch                string = "jsfetch"
+	BenchTypeOldJSOrdered           string = "oldjsordered"
+	BenchTypeOldJSPush              string = "oldjspush"
+	BenchTypeOldJSPull              string = "oldjspull"
+	BenchTypeKVPut                  string = "kvput"
+	BenchTypeKVGet                  string = "kvget"
+	AckModeNone                     string = "none"
+	AckModeAll                      string = "all"
+	AckModeExplicit                 string = "explicit"
```

Comment:
no biggie but `string` in all of these are not needed.

Reply:
Cleaned up